### DISCUSSION TITLE
Any schedule for the next release?

### DISCUSSION THREAD

**Discussion by skj-skj** [aniyomiorg/aniyomi#1209] [Q&A]
Hi, I see the dev is working and made commits regularly and preview builds are released often. I would like to know is there any plan for next stable release because last stable release was in feb.
if the answer is already in other post or issue. please send the link to that discussion.

Thanks

**Comment by Nintenstation** [comment]
> Hi, I see the dev is working and made commits regularly and preview builds are released often. I would like to know is there any plan for next stable release because last stable release was in feb.
> if the answer is already in other post or issue. please send the link to that discussion.
> 
> Thanks
> 

I'm also curious about that. I know they're busy and I respect their time on it, but it seems like the "stable" version is anything but stable as of now lol.
